api/mongo.py

```diff
@@ -0,0 +1,26 @@
+import pymongo
+import datetime
+
+db = None
+
+
+def configure_db(db_uri, site_id, site_name, api_uri):
+    global db
+    db = pymongo.MongoClient(db_uri).get_default_database()
```

Comment:
good point.
I will add a try catch block similar to the one that existed before.